### PR TITLE
Vagabond drip DLC, or in proper words, Style vendor

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

Adds a new vendor with VERY stylish clothing

also added newline for voidsuits cause there wasn't one earlier

image

## Why It's Good For The Game

It's not

Requested by Grey on discord, also option to get stylish clothing in game

## Changelog
:cl: Code - Wouju, Sprites - InfraredBaron
add: Added new stylish clothing in a stylish vendor in a stylish place
imageadd: added new stylish icons for stylish clothes that are contained within a stylish vendor
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
powercreep.

**Comment by TheShown911** [comment]
airplane.

**Comment by Wouju** [comment]
aight since you fucks saw this, thoughts on the armour values of some of the clothing and their description?

**Comment by Sparrowbird611** [comment]
> aight since you fucks saw this, thoughts on the armour values of some of the clothing and their description?

1. I think you should rename the PR since it's misleading.
2. Everything is two expensive for near useless equipment.
3. Armor values are all up to you to be frank, none of these seem terrible since they are all below average actual armor. For what you charge, you could do more.

**Comment by Wouju** [comment]
1. literally 1984
2. the main "appeal" is the high style, it's not exactly something that you should easily get in large amounts
3. refers to point 2, the armour is there so it's not always just a different sprite, and for the suits so they aren't piss useless against a plastic knife

**Comment by Omni44** [comment]
Honestly, none of this needs any actually good armor values. Throw some bio at raincoat, rebreather and the shoes, maybe some minor resistance on skulls and that's it.

**Comment by Sparrowbird611** [comment]
> 1. literally 1984
> 2. the main "appeal" is the high style, it's not exactly something that you should easily get in large amounts
> 3. refers to point 2, the armour is there so it's not always just a different sprite, and for the suits so they aren't piss useless against a plastic knife

Bottom line is style isn't worth more than a good gun or good protection. You can keep it semi-expensive, but reduce it to a more reasonable price, as only the artist is realistically purchasing this stuff consistently. 400-500 range isn't a bad chunk of cash for slightly faster insight gain.

**Comment by Wouju** [comment]
frankly i still don't see the exact problem

yes the clothing is overpriced, with the only appeal being high style, but i think that's kinda the point

if you want armour proper there's the serb vendor with its flak crates

if you want ridiclously expensive stylish clothing there's this vendor

**Comment by Sparrowbird611** [comment]
Also to nitpick a bit, on the ruiner reference jacket, you say it's "well armored" in the description. Remove that or give it at least slightly above basic armor stats.

**Comment by Wouju** [comment]
> Also to nitpick a bit, on the ruiner reference jacket, you say it's "well armored" in the description. Remove that or give it at least slightly above basic armor stats.

fair

**Comment by Sparrowbird611** [comment]
> frankly i still don't see the exact problem
> 
> yes the clothing is overpriced, with the only appeal being high style, but i think that's kinda the point
> 
> if you want armour proper there's the serb vendor with its flak crates
> 
> if you want ridiclously expensive stylish clothing there's this vendor

Up to you to be frank, I think it's stupid to make what I frankly consider near useless stuff this expensive.

**Comment by Wouju** [comment]
> At least you'll make a pretty corpse though.

if i'm going down i'm going down in style

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -254,6 +254,78 @@ obj/item/clothing/under/genericr
...
+		rad = 0
+	)
+
+/obj/item/clothing/under/style/johnny/raider
+	name = "leather outfit"
+	desc = "A rather generic leather outfit. Smells like waste"
+	icon_state = "raider"
+	item_state = "raider"
+
+/obj/item/clothing/under/style/johnny/helltaker
```
Comment (GrayRachnid) [inline code/modules/clothing/under/miscellaneous.dm:323]: why is this a child of johnny

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -260,3 +260,28 @@
...
 	)
 
+/obj/item/clothing/head/metaljacket
+	name = "sergeant hat"
+	desc = "A rather generic sergeant-like hat."
+	icon_state = "metaljacket"
+	item_state = "metaljacket"
+	price_tag = 400
+
+/obj/item/clothing/head/inhailer
```
Comment (GrayRachnid) [inline code/modules/clothing/head/misc.dm:270]: inhaler*

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1561,6 +1561,53 @@
...
+		/obj/item/clothing/under/style/dress = 600,
+		/obj/item/clothing/under/style/dress/white = 600,
+		/obj/item/clothing/under/style/johnny = 750,
+		/obj/item/clothing/under/style/johnny/raider = 750,
+		/obj/item/clothing/under/style/johnny/helltaker = 750,
+		/obj/item/clothing/suit/storage/triad = 1200,
+		/obj/item/clothing/suit/storage/akira = 750,
+
+					)
+	vendor_department = DEPARTMENT_OFFSHIP
```
Comment (GrayRachnid) [inline code/game/machinery/vending.dm:1609]: guild

File: code/modules/clothing/head/misc.dm [hunk 4]
```diff
@@ -260,3 +260,28 @@
...
+	name = "odd looking helmet"
+	desc = "A confusingly complex helmet. It is capable of protecting you so it's more useful than being a simple decoration."
+	icon_state = "inhailer"
```
Comment (GrayRachnid) [inline code/modules/clothing/head/misc.dm:273]: fix these too

File: code/modules/clothing/under/miscellaneous.dm [hunk 5]
```diff
@@ -254,6 +254,78 @@ obj/item/clothing/under/genericr
 	style = STYLE_HIGH
 	price_tag = 300
 
+//stylish vendor clothes
```
Comment (GrayRachnid) [inline code/modules/clothing/under/miscellaneous.dm:257]: are you seriously so lazy to add style = style_high that you made them all a child of this?

bad practice. no.

**Comment by GrayRachnid** [comment]
I'm fine with the prices as they are. Just make sure the money goes to the guild. The prices are too negligible(compared to FS guns) for this to be an offship vendor.

If guild makes money off of this, good for them.

File: code/modules/clothing/under/miscellaneous.dm (hunk 5 above)
Comment (Wouju) [reply code/modules/clothing/under/miscellaneous.dm:257]: > are you seriously so lazy to add style = style_high that you made them all a child of this?

yes

~~gonna change that, literally 1984~~

**Comment by granodd** [comment]
Prices are fine. Have you ever heard of the real life fashion industry?

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (Wouju) [reply code/modules/clothing/under/miscellaneous.dm:323]: my bad, must have accidentaly copied it

**Comment by Wouju** [comment]
> Prices are fine. Have you ever heard of the real life fashion industry?

keked

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (Wouju) [reply code/modules/clothing/under/miscellaneous.dm]: done, now its like the other normal uniforms

File: code/modules/clothing/under/miscellaneous.dm (hunk 5 above)
Comment (Wouju) [reply code/modules/clothing/under/miscellaneous.dm:257]: done

File: code/modules/clothing/head/misc.dm (hunk 4 above)
Comment (Wouju) [reply code/modules/clothing/head/misc.dm]: done, also changed the sarge hat into a ranger hat

File: code/game/machinery/vending.dm (hunk 3 above)
Comment (Wouju) [reply code/game/machinery/vending.dm]: done

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (Wouju) [reply code/modules/clothing/head/misc.dm]: done

File: code/game/machinery/vending.dm [hunk 6]
```diff
@@ -1561,6 +1561,53 @@
...
+		/obj/item/clothing/under/dress = 600,
+		/obj/item/clothing/under/dress/white = 600,
+		/obj/item/clothing/under/helltaker = 600,
+		/obj/item/clothing/under/johnny = 750,
+		/obj/item/clothing/under/raider = 750,
+		/obj/item/clothing/suit/storage/triad = 1200,
+		/obj/item/clothing/suit/storage/akira = 750,
+
+					)
+	vendor_department = DEPARTMENT_GUILD
```
Comment (GrayRachnid) [inline code/game/machinery/vending.dm:1609]: You might not actually need this since all vendors default to guild. Double check though

Comment (Wouju) [reply code/game/machinery/vending.dm:1609]: image

Comment (zombiemimic) [reply code/game/machinery/vending.dm:1609]: yeah, they do

**Review by Firefox13 — COMMENTED** [review]
A lot, almost all of these items can go into loadout.
Please don't add style values onto gloves/boots because stacking it will lead to pretty OVERWHELMING(you dig me??) behaviour.

**Comment by GrayRachnid** [comment]
> A lot, almost all of these items can go into loadout.
> Please don't add style values onto gloves/boots because stacking it will lead to pretty OVERWHELMING(you dig me??) behaviour.

None of the clothes in loadout have high style. These can be obtained using ingame dosh for their high style perk. 

Also, if we don't add high style to other times we will never be able to get max style

**Comment by TheShown911** [comment]
Jesus Christ put a picture of the cool sprites in the PR this is a war crime.

**Comment by Sparrowbird611** [comment]
Technically, if we go off the original style document, the loadout clothing pieces are supposed to all have a style value of 2 to encourage people wearing different clothing items, so unless we have some kind of balance issue with that, we could just do what firefox said and add a style boost to clothing.

**Comment by Wouju** [comment]
> Jesus Christ put a picture of the cool sprites in the PR this is a war crime.

never

~~gimme some time~~

**Comment by Sparrowbird611** [comment]
> > A lot, almost all of these items can go into loadout.
> > Please don't add style values onto gloves/boots because stacking it will lead to pretty OVERWHELMING(you dig me??) behaviour.
> 
> None of the clothes in loadout have high style. These can be obtained using ingame dosh for their high style perk.
> 
> Also, if we don't add high style to other times we will never be able to get max style

After looking at the loadout, this is mostly false. Most suits and hats in the loadout have high style, and positive style values are rarely applied to undersuits, gloves, and shoes.

**Comment by GrayRachnid** [comment]
> and positive style values are rarely applied to undersuits, gloves, and shoes.

More reason to put them on these items for a price

**Comment by Sparrowbird611** [comment]
> > and positive style values are rarely applied to undersuits, gloves, and shoes.
> 
> More reason to put them on these items for a price
> 
> Not against adding them to loadout but I feel like they should still be obtainable IC. Why is everything locked to loadout?

Done does plan to add a clothing vendor. Perhaps talk to them about consolidating these into that

**Comment by Wouju** [comment]
>Maybe you should expand this version to include all clothing ;). I think that'd be pretty based.

suffer for all eternity as i refuses to do that

**Comment by Wouju** [comment]
https://github.com/discordia-space/CEV-Eris/pull/6355

new pr cause i cant unfuck this one i hate DMI

**Pull request closed without merging**